packages/connect-popup/e2e/tests/browser-support.test.ts

```diff
@@ -40,7 +40,7 @@ test('unsupported browser', async ({ browser }) => {
     popup = await openPopup(page);
     await popup.waitForSelector('text=Unsupported browser');
     await popup.screenshot({ path: `${dir}/browser-not-supported.png` });
-    await popup.close();
+    await popup.close({ runBeforeUnload: true });
```

Comment:
curiosity here. why do we need this. this is this option right? https://playwright.dev/dotnet/docs/dialogs#beforeunload-dialog

Reply:
Yes, it is optional for playwright. But we use it as listener for webextension https://github.com/trezor/trezor-suite/blob/develop/packages/connect-webextension/src/contentScript.ts#L46 and if not set in the test then it is not triggered. This specific test `browser-support.test` is not tested with webextension and probably does not make sense but I thought it would be better to be consistent in the way we use close in the test.

Reply:
makes sense.